README.md

```diff
@@ -1,58 +1,42 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
+## Major Dependencies
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker
+  - OSX recommended install method: [Download from Docker website](https://docs.docker.com/mac/)
+- docker-compose
+  - Recommended install: pip `pip install docker-compose`
 
-Although you can run Micromasters locally with a default sqlite database after
-installing the ``requirements.txt`` file, the recommended way is to 
-use [Docker](https://www.docker.io). Install Docker, then install 
-``docker-compose`` and run the app: 
+## (OSX only) Getting your machine Docker-ready
 
-    pip install docker-compose 
-    docker-compose up
+#### Create your docker container:
 
-### (OSX only) Create your docker machine:
-
-    docker-machine create default
-    docker-machine start default
-    docker-machine env default
-    eval "$(docker-machine env default)"
-
-These commands create a Docker container named ``default``, start the
+The following commands create a Docker machine named ``mm``, start the
 container, and configure environment variables to facilitate communication
-with the ``edX`` instance.
-
-
-This will set up a near production-ready containerized development 
-environment that runs migrations, with the Django development server 
-running on port ``8079``.
-
-To run one-off commands, like shell, you can run:
-
-    docker-compose run web python manage.py shell
+with the edX instance.
 
-or to create root user:
+    docker-machine create mm
```

Comment:
i _knew_ i was forgetting something. i had a note about this and it got lost in the pile. i'll add this (and add virtualbox to the major dependencies)